[2016-09-20 08:54:35,328] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
[2016-09-20 09:40:45,652] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71efol80c
frameImg.tuiguang.png action not found

Action not found
Action frameImg.tuiguang.png could not be found. Error raised is Controller controllers.frameImg.tuiguang not found

play.exceptions.ActionNotFoundException: Action frameImg.tuiguang.png not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:566)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: Controller controllers.frameImg.tuiguang not found
	... 3 more
[2016-09-20 10:17:20,361] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-20 10:18:48,823] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71efol80d
tasks.list action not found

Action not found
Action tasks.list could not be found. Error raised is Controller controllers.tasks not found

play.exceptions.ActionNotFoundException: Action tasks.list not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:566)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: Controller controllers.tasks not found
	... 3 more
[2016-09-20 10:19:11,598] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [vo] Required
[2016-09-20 10:52:32,405] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-20 10:52:32,405] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
java.lang.NullPointerException
	at models.BuyerTask2.findPcCountByPlatform(BuyerTask2.java:1381)
	at controllers.TaskCenter.deviceCount2(TaskCenter.java:291)
	at sun.reflect.GeneratedMethodAccessor701.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-20 10:52:32,527] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-20 10:52:32,527] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
java.lang.NullPointerException
	at controllers.TaskCenter.taskTypeCount2(TaskCenter.java:376)
	at sun.reflect.GeneratedMethodAccessor702.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-20 10:52:37,367] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-20 10:52:37,367] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
java.lang.NullPointerException
	at controllers.TaskCenter.taskTypeCount2(TaskCenter.java:376)
	at sun.reflect.GeneratedMethodAccessor702.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-20 10:52:37,465] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-20 10:52:37,465] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
java.lang.NullPointerException
	at models.BuyerTask2.findPcCountByPlatform(BuyerTask2.java:1381)
	at controllers.TaskCenter.deviceCount2(TaskCenter.java:291)
	at sun.reflect.GeneratedMethodAccessor701.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-20 10:52:40,730] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-20 10:52:40,730] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
java.lang.NullPointerException
	at models.BuyerTask2.findPcCountByPlatform(BuyerTask2.java:1381)
	at controllers.TaskCenter.deviceCount2(TaskCenter.java:291)
	at sun.reflect.GeneratedMethodAccessor701.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-20 10:52:40,815] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:232) - =======>Server Fatal Error<======
[2016-09-20 10:52:40,815] ERROR [play-thread-1] com.aton.base.BaseController.handleServerError(BaseController.java:233) - 
java.lang.NullPointerException
	at controllers.TaskCenter.taskTypeCount2(TaskCenter.java:376)
	at sun.reflect.GeneratedMethodAccessor702.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:524)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:475)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:451)
	at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:446)
	at play.mvc.ActionInvoker.invoke(ActionInvoker.java:160)
	at play.server.PlayHandler$NettyInvocation.execute(PlayHandler.java:267)
	at play.Invoker$Invocation$1.apply(Invoker.java:290)
	at play.Invoker$Invocation$1.apply(Invoker.java:287)
	at play.db.jpa.JPA.withTransaction(JPA.java:258)
	at play.db.jpa.JPA.withinFilter(JPA.java:217)
	at play.db.jpa.JPAPlugin$TransactionalFilter.withinFilter(JPAPlugin.java:299)
	at play.Invoker$Invocation.withinFilter(Invoker.java:271)
	at play.Invoker$Invocation.run(Invoker.java:287)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:245)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:98)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:206)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[2016-09-20 10:58:27,044] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-09-20 10:58:38,481] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: Incorrect value
[2016-09-20 10:58:41,800] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [type] Required
[2016-09-20 10:59:04,737] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [type] Required
[2016-09-20 11:01:20,286] ERROR [play-thread-1] com.aton.base.BaseController.handleWrongInput(BaseController.java:291) - Input wrong: [type] Required
[2016-09-20 11:07:38,304] ERROR [play-thread-1] play.Logger.niceThrowable(Logger.java:604) - 

@71efol80e
buyer.task action not found

Action not found
Action buyer.task could not be found. Error raised is Controller controllers.buyer not found

play.exceptions.ActionNotFoundException: Action buyer.task not found
	at play.mvc.ActionInvoker.getActionMethod(ActionInvoker.java:566)
	at play.mvc.ActionInvoker.resolve(ActionInvoker.java:84)
	at Invocation.HTTP Request(Play!)
Caused by: java.lang.Exception: Controller controllers.buyer not found
	... 3 more
[2016-09-20 13:42:32,659] ERROR [play-thread-1] com.aton.base.secure.Secure.checkAuthentication(Secure.java:92) - Login admin not exists,admin_id=null
